stepup/tests/behat/features/selfservice_activation_flow.feature

```diff
@@ -0,0 +1,46 @@
+Feature: A user manages his tokens in the SelfService portal
+  In order to use a second factor token
+  As a user
+  I must be able to manage my second factor tokens
+
+  Scenario: A user registers a Yubikey token in selfservice using RA vetting preference
+  Given I receive the following attributes for "joe-a4" from the IdP:
```

Comment:
Awesome test!!

Suggestion: maybe create a dedicated step definition to indicate you expect the IdP to release no attributes? Perhaps a bit more expressive than the empty table?